tools/kochdocs.nim

```diff
@@ -318,6 +318,9 @@ proc buildDocsDir*(args: string, dir: string) =
   buildDocPackages(args, dir)
   copyFile(docHackJsSource, dir / docHackJsSource.lastPathPart)
 
-proc buildDocs*(args: string) =
-  buildDocsDir(args, webUploadOutput / NimVersion)
-  buildDocsDir("", docHtmlOutput) # no `args` to avoid offline docs containing the 'gaCode'!
+proc buildDocs*(args: string, localOnly = false, localOut = "") =
```

Comment:
nit: code smell: localOut is only used for localOnly = true; the previous way (when `buildDocsDir` was called directly) was simpler